config.toml

```diff
@@ -128,9 +128,34 @@ rpc_addr = 'http://127.0.0.1:26657'
 # Specify the GRPC address and port where the chain GRPC server listens on. Required
 grpc_addr = 'http://127.0.0.1:9090'
 
-# Specify the WebSocket address and port where the chain WebSocket server
-# listens on. Required
-websocket_addr = 'ws://127.0.0.1:26657/websocket'
+# The type of event source to use for getting events from the chain.
+#
+# This setting can take two types of values, as an inline table:
+# 
+# a) Push: for receiving IBC events over WebSocket
+#
+#      `{ mode = 'push', url = 'ws://127.0.0.1:26657/websocket', batch_delay = '500ms' }`
+#
+#    where
+#
+#    - `url` is the WebSocket URL to connect to. Required
+#    - `batch_delay` is the delay until event batch is
+#      emitted in the absence of NewBlock event. Default: 500ms
+#      Lower values will result in faster event processing, improving the latency of Hermes,
+#      but may split the events into more batches than necessary, requiring more client updates
+#      to be submitted, yielding higher costs. Higher values will result in slower event
+#      processing, increasing the latency of Hermes, but are more likely to batch events together.
+#      The default value provides good latency while minimizing the number of client updates needed.
+
+# b) Poll: for polling for IBC events via the `/block_results` RPC endpoint
+#
+#     `{ mode = 'poll', poll_interval = '1s' }`
```

Comment:
Change to conform with code:
```suggestion
#     `{ mode = 'pull', poll_interval = '1s' }`
```
But wondering if this would cause confusion with `pull` in `mode` and `poll_..` for the interval and should consider using `poll` everywhere.

Reply:
682efd2b